src/server/rest/v1/service/security/RegistrationTokenSecurity.ts

```diff
@@ -23,6 +23,7 @@ export default class RegistrationTokenSecurity {
 
   public static filterRegistrationTokenRequest(request: any): Partial<RegistrationToken> {
```

Comment:
Create a new type here like HttpSiteAreaRequest
And GET params are in PascalCase: ID, Description, SiteAreaID.
Moreover for this get I would expect only the ID to get one token.

Reply:
Modified as `filterRegistrationTokenByIDRequest` as we only need the id in the end